rbc/tests/heavyai/test_text_encoding.py

```diff
@@ -227,6 +227,7 @@ def test_text_encoding_column_list2(heavydb, size):
 @pytest.mark.usefixtures("create_columns")
 @pytest.mark.parametrize("size", (32,))
 @pytest.mark.parametrize("num_cols", (1, 2, 3, 4))
+@pytest.mark.skip()
```

Comment:
Leftover? If not make sure to keep track of it somewhere.

Reply:
There's a bug in HeavyDB master which crashes the server when running this test. I'll open an issue to investigate it.

Reply:
ok thanks